ios/FluentUI/Controls/ButtonVnext.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+public class ButtonVnextState: ObservableObject {
+	@Published var title: String = ""
+	@Published var isDisabled: Bool = false
+}
+
+public struct ButtonVnextViewButtonStyle: ButtonStyle {
```

Comment:
Won't there be a retain cycle if this knows about the button but the button gets this reference as its button style? In examples I've seen I haven't seen the ButtonStyle have to know about the button. Can we prevent that here and get the tokens be on the style instead of require the tokens to be passed in to create the style?

Reply:
You're right. I'm now passing the tokens and state objects to the Style and not the button anymore.